### ISSUE TITLE
Public Token Incorrectly Referred To As "Access Token"

### ISSUE THREAD

**Issue by IsaiahJTurner** [simonlevy5/plaid-ios-link#6]
The Plaid link API returns a public token but within the `plaid-ios-link` code it is incorrectly referred to as an access token. This can cause confusion.

**Comment by northoutvivek** [comment]
Yes, Did you get any clue on this, even using sdk but not getting access token as well.

 func linkNavigationContoller(_ navigationController: PLDLinkNavigationViewController!, didFinishWithAccessToken accessToken: String!) {
}
